src/main/java/gregtech/client/renderer/texture/cube/OrientedOverlayRenderer.java

```diff
@@ -144,6 +155,10 @@ public void registerIcons(TextureMap textureMap) {
             sprites.put(overlayFace, new ActivePredicate(normalSprite, activeSprite, pausedSprite,
                     normalSpriteEmissive, activeSpriteEmissive, pausedSpriteEmissive));
         }
+
+        if (!foundTexture) {
+            FMLClientHandler.instance().trackMissingTexture(new ResourceLocation(modID, "blocks/" + basePath + "/overlay_OVERLAY_FACE"));
```

Comment:
I am trying to figure out if there is some better way here to notify the user that we did not find any overlay face for the block, because I don't really like the use of `OVERLAY_FACE`, but I cannot think of anything. Its minor enough that it should not block the PR, but if you can think of a better way, I feel like it could be improved.